renku/cli/dataset.py

```diff
@@ -104,6 +104,26 @@
         new-dir/
           new-filename
 
+Updating a dataset:
+
+After adding files from a remote Git repository, you can check for updates in
+those files by using ``renku dataset update`` command. This command checks all
+remote files and copies over new content if there are any. It also deletes
+files from the local dataset if they are deleted from the remote Git
+repository.
+
+You can limit the scope of updated files by specifying dataset names, using
+``--include`` and ``--exclude`` to filter based on file names, or using
+``--creators`` to filter based on creators. For example, the following command
```

Comment:
creators - as determined by the email?

Reply:
No, it is the full name of the creator. This is like some other dataset subcommands (e.g. `ls-files`)